apps/docs/pages/guides/ai/vector-indexes/ivf-indexes.mdx

```diff
@@ -1,90 +1,99 @@
 import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
-  id: 'ai-managing-indexes',
-  title: 'Managing indexes',
-  description: 'Understanding vector indexes',
-  sidebar_label: 'Managing indexes',
+  id: 'ai-ivf-indexes',
+  title: 'IVFFlat indexes',
+  description: 'Understanding IVFFlat indexes in pgvector',
+  sidebar_label: 'IVFFlat indexes',
 }
 
-Once your vector table starts to grow, you will likely want to add an index to speed up queries. Without indexes, you'll be performing a sequential scan which can be a resource-intensive operation when you have many records.
+IVFFlat is a type of vector index for approximate nearest neighbor search, used in high-dimensional spaces like those found in embeddings.
 
-## IVFFlat indexes
+## Choosing an index
 
-Today `pgvector` indexes use an algorithm called IVFFlat. IVF stands for 'inverted file indexes'. It works by clustering your vectors in order to reduce the similarity search scope. Rather than comparing a vector to every other vector, the vector is only compared against vectors within the same cell cluster (or nearby clusters, depending on your configuration).
+Today `pgvector` supports two types of indexes:
 
-### Inverted lists (cell clusters)
+- [HNSW](/docs/guides/ai/vector-indexes/hnsw-indexes)
+- [IVFFlat](/docs/guides/ai/vector-indexes/ivf-indexes)
 
-When you create the index, you choose the number of inverted lists (cell clusters). Increase this number to speed up queries, but at the expense of recall.
+In general we recommend using [HNSW](/docs/guides/ai/vector-indexes/hnsw-indexes) because of its [performance gains](https://supabase.com/blog/increase-performance-pgvector-hnsw#hnsw-performance-1536-dimensions) and [robustness against changing data](/docs/guides/ai/vector-indexes/hnsw-indexes#when-should-you-create-hnsw-indexes). If you have a special use case that requires IVFFlat instead, keep reading.
```

Comment:
```suggestion
In general we recommend using [HNSW](/docs/guides/ai/vector-indexes/hnsw-indexes) because of its [performance](https://supabase.com/blog/increase-performance-pgvector-hnsw#hnsw-performance-1536-dimensions) and [robustness against changing data](/docs/guides/ai/vector-indexes/hnsw-indexes#when-should-you-create-hnsw-indexes). If you have a special use case that requires IVFFlat instead, keep reading.
```